pkgs/misc/cups/drivers/fflinuxprint/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, lib, fetchurl, dpkg, autoPatchelfHook, cups }:
+let
+  debPlatform =
+    if stdenv.hostPlatform.system == "x86_64-linux" then "amd64"
+    else if stdenv.hostPlatform.system == "i686-linux" then "i386"
+         else throw "Unsupported system: ${stdenv.hostPlatform.system}";
+in
+stdenv.mkDerivation rec {
+  pname = "fflinuxprint";
+  version = "1.1.3-4";
+
+  src = fetchurl {
+    url = "https://support-fb.fujifilm.com/driver_downloads/fflinuxprint_1.1.3-4_amd64.deb";
+    sha256 = "434a81e20bc459ad74286b7a4a7815aab6b115e3f1210eb69d3ac5678e2fcab5";
+    curlOpts = "--user-agent Mozilla/5.0";  # HTTP 410 otherwise
+  };
+
+  nativeBuildInputs = [ dpkg autoPatchelfHook ];
+  buildInputs = [ cups ];
+
+  sourceRoot = ".";
+  unpackCmd = "dpkg-deb -x $curSrc .";
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    mv etc $out
+    mv usr/lib $out
+    mkdir -p $out/share/cups/model
+    mv usr/share/ppd/fujifilm/* $out/share/cups/model
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "FujiFILM Linux Printer Driver";
+    longDescription = ''
```

Comment:
I don't think we should get rid of all the supported devices or else we'll have to maintain it as well.

Reply:
I've removed the details of the supported devices.